pages/apiPagesNegFlow/loginNegFlow_testCases.ts

```diff
@@ -0,0 +1,263 @@
+import { apiDataSet } from "../../utils/dataSet"
+
+export const loginGuestTestCases = [
```

Comment:
@YulianH change page and import name from TC to payload

Reply:
done